### PR TITLE
Eskip patch filters

### PR DESCRIPTION
**Pull request by aryszka**

### CODE REVIEW

File: etcd/etcd_test.go [hunk 1]
```diff
@@ -15,10 +15,10 @@
 package etcd
 
 import (
+	"errors"
```
Comment (danpersa) [inline etcd/etcd_test.go]: how is that the order got changed?

Comment (aryszka) [inline etcd/etcd_test.go]: no idea... but this looks now alphabetical

**Comment by danpersa** [comment]
lgtm

File: cmd/eskip/args.go [hunk 2]
```diff
@@ -192,6 +205,28 @@ func processStdin() (*medium, error) {
...
+	if pfile != "" {
+		media = append(media, &medium{typ: patchPrependFile, patchFile: pfile})
+	}
+
+	if afile != "" {
+		media = append(media, &medium{typ: patchAppendFile, patchFile: afile})
+	}
+
+	return media, nil
+}
```
Comment (lmineiro) [inline cmd/eskip/args.go]: This doesn't seem to ever error, why the return error in the signature?

File: cmd/eskip/doc.go [hunk 3]
```diff
@@ -90,6 +94,14 @@ stdin         standard input when not tty, expecting routes
 file          a file containing routes
 inline        routes as command line parameter
 inline ids    a list of route ids (only for delete)
+prepend       a chain of filters to be prepended to the filter chain in
+              each route
+prepend file  a file containing a chain of filters to be prepended to the
+              filter chain in each route
+append        a chain of filters to be prepended to the filter chain in
+              each route
+append file   a file containing a chain of filters to be prepended to the
```
Comment (lmineiro) [inline cmd/eskip/doc.go]: "to be appended": s/prepended/appended

File: cmd/eskip/doc.go [hunk 4]
```diff
@@ -90,6 +94,14 @@ stdin         standard input when not tty, expecting routes
 file          a file containing routes
 inline        routes as command line parameter
 inline ids    a list of route ids (only for delete)
+prepend       a chain of filters to be prepended to the filter chain in
+              each route
+prepend file  a file containing a chain of filters to be prepended to the
+              filter chain in each route
+append        a chain of filters to be prepended to the filter chain in
```
Comment (lmineiro) [inline cmd/eskip/doc.go]: "to be appended": s/prepended/appended

File: cmd/eskip/args.go (hunk 2 above)
Comment (aryszka) [inline cmd/eskip/args.go]: true

**Pull request merged**